android/app/build.gradle

```diff
@@ -179,6 +192,19 @@ android {
     }
 
     // applicationVariants are e.g. debug, release
+
+    flavorDimensions 'type'
+    productFlavors {
+        foss {
+            dimension = 'type'
+            buildConfigField 'boolean', 'PLAY_BUILD', "false"
+        }
+        play {
+            dimension = 'type'
+            buildConfigField 'boolean', 'PLAY_BUILD', "true"
+        }
+    }
```

Comment:
The default is to use `"` in this file.

Reply:
Need more light on this. Pls explain more

Reply:
It's just to change the `single quotes (')` for `double quotes (")`. :)

Reply:
Oh, Thanks will update soon :)